### ISSUE TITLE
Can't launch app on iphone

### ISSUE THREAD

**Issue by digitalplaywright** [jhabdas/lumpen-radio#1]
I tried downloading your app on the app store and it did not work on my iphone 5 with ios 9.0 (launches and then shuts down instantaneously). I get another error when compiling master: `STKDataSource.h` file not found. Copying in the file only seems to get me to a second error in linking.

Please let me know if you have any tips on how to solve these errors. I am very curious to see your approach live. :)

**Comment by jhabdas** [comment]
Thanks for the report, @digitalplaywright. Yesterday I reworked the app to support iOS 9, the latest version of React Native submitted within the last week and also integrated Webpack Dev Server. Hopefully the journey through the App Store for the iOS 9 build I submitted will be a swift one. Please stay tuned. Meanwhile please tune in on the Web at www.lumpenradio.com as a workaround. We'll keep this issue open until support is confirmed.

**Comment by digitalplaywright** [comment]
Thanks! I still get the 'STKDataSource.h' error when building the source in XCode. My node version is v4.0.0 and pod 0.38.2. I see that the Podfile specifies ios 8.0 so maybe that is a contributing factor to the error, but the error remains after I change that to 9.0. Any suggestions on how to build the source?

**Comment by jhabdas** [comment]
No problem. The app is built to work with iOS 8 forward, so both in the `Podfile` and `Info.plist` it shouldn't be surprising to see iOS 8 stuff. And it shouldn't need to be changed to build the app locally.

That `STKDataSource.h` not found issue I've experienced before, even before iOS 9 was out, while setting my project up on my own machine. I believe it may be a CocoaPods issue, but it's hard to say without more information. Would you be able to remove the `Pods` folder from your install, do another `pod install` and let me know if you receive any *warnings* during Pod install with your current version of CocoaPods?

**Comment by jhabdas** [comment]
Quick update to my [earlier comment](https://github.com/jhabdas/lumpen-radio/issues/1#issuecomment-141751787) on fix submitted to the App Store. The build submitted to fix the iOS 9 bug is still `Waiting For Review`.

**Comment by jhabdas** [comment]
Updating app for Xcode 7, React 0.11.0 and the new CocoaPods didn't unblock me from getting updates into the store. As a result, I've made some additional updates to fix some warnings in the app and submitted another build to Apple. If anyone's into reading crash logs from Apple hit me up.

**Comment by jhabdas** [comment]
Lumpen Radio 1.4 just hit App Store with better iOS 9 support and a base for an Android app. If you have any other issues please feel free to open a new issue.
